### PR TITLE
Automatically post PR Looms to Slack

### PR DESCRIPTION
**Pull request by fregante**
To do:

- [x] Add workflow to remind users after a positive review
- [x] Double-check [slack action](https://github.com/marketplace/actions/slack-notify)
- [ ] Add Slack hook secret: @twschiller 

https://www.loom.com/share/123456789a123456789a123456789a12

### CODE REVIEW

File: .github/workflows/loom-slack.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Loom to slack
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  post:
+    if: github.event.pull_request.merged == 'true'
```
Comment (twschiller) [inline .github/workflows/loom-slack.yml:10]: 🥳

File: .github/workflows/loom-reminder.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+            After the PR is merged, this loom will be posted to `#sprint-demo` on Slack:
+
+            ${{ env.LOOM_LINK }}
+
+      - name: Remind to post loom if missing
+        uses: marocchino/sticky-pull-request-comment@v2
+        if: ${{ ! env.LOOM_LINK }}
+        with:
+          message: |
+            No loom links were found in the first post. Please add one if you'd like to it to appear on Slack.
```
Comment (fregante) [inline .github/workflows/loom-reminder.yml:36]: The comment will be updated after every review. In the future, [if supported](https://github.com/marocchino/sticky-pull-request-comment/issues/783), I'll add a similar workflow to automatically update the comment every time the first post is updated

**Review by fregante — COMMENTED** [review]
This should be good to go, it just needs a secret and a production run. I will also post a link to these workflows in the app once verified.

**Comment by twschiller** [comment]
@fregante there's already a secret called SLACK_WEBHOOK. Is that not working? I guess we'll find out

**Review by twschiller — APPROVED** [review]

**Comment by fregante** [comment]
Oh great! I don’t see what secrets are available. It also seems to be working on this PR already, so I’ll add a link and then merge

**Comment by fregante** [comment]
It seems to have failed on the merge condition, 🥲 I thought I had verified it. 

https://github.com/pixiebrix/pixiebrix-extension/blob/107d6ce52d82187ad407100051e91c2ba5722841/.github/workflows/loom-slack.yml#L11

**Pull request merged**